spec/routing/hq/sponsors_routing_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Hq::Sponsors routing', type: :routing do
```

Comment:
I much prefer the style and readability that you used for partners and would prefer to stick to just one style.